src/js/git/commands.js

```diff
@@ -923,23 +923,43 @@ var commandConfig = {
     regex: /^git +switch($|\s)/,
     options: [
       '-c',
+      '--create',
+      '-C',
+      '--force-create',
       '-'
     ],
     execute: function(engine, command) {
       var generalArgs = command.getGeneralArgs();
       var commandOptions = command.getOptionsMap();
 
-      var args = null;
-      if (commandOptions['-c']) {
-        // the user is really trying to just make a
-        // branch and then switch to it. so first:
-        args = commandOptions['-c'].concat(generalArgs);
-        command.twoArgsImpliedHead(args, '-c');
+      {
```

Comment:
is there a reason for this open bracket? sorry maybe this is some new JS syntax I am unfamiliar with?

is it just scoping the `createOption` variable?

Reply:
> is it just scoping the createOption variable?

Yes. It is just for scoping.  
I think the outer scope here would be growing, so I want to keep these temporary variables short lived.

It is completely opinion based thing, so I don't care if you want to remove them.

Reply:
That's also why I changed some variables `var` to `let`. `let` variables are removed when they go out of brackets, but `var` ones are still alive. 

https://developer.mozilla.org/en-US/docs/Glossary/Scope

Reply:
Ah I learned something today then 😇 thanks for the explanation. At Meta we don't really do this kind of sub function scoping so this is the first I've seen of it.

I started this project over ten years ago and sadly let and const weren't in the language yet, so that's why everything is var. A ton of this code could be a lot cleaner if it was rewritten today...
M